### PR TITLE
Add Stretch blueprint

### PR DESCRIPTION
**Pull request by heucke**
Signed-off-by: Tyler Heucke <tyler.heucke@dell.com>

### CODE REVIEW

**Comment by heucke** [comment]
Pull-request updated, HEAD is now 9602e24b9dfd08527b4ca9e233d0a3bc0898254e

**Comment by heucke** [comment]
Pull-request updated, HEAD is now 55abb504d290f6ae30d34adef163f3c65374d76f

**Comment by heucke** [comment]
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=847986;msg=10

We'll have to upgrade the Docker image from Jessie to Stretch.

**Comment by heucke** [comment]
Pull-request updated, HEAD is now 3d58e49fb30d486a405a6a8c3b5f6e0ffc88e57a

**Comment by heucke** [comment]
Pull-request updated, HEAD is now 0b178707cdf8e93f36aff8abc2aba247f501908c

File: release_bp/opx_debian_base_stretch.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<package_set xmlns:xi="http://www.w3.org/2001/XInclude">
+  <!-- Packages to be installed on all targets -->
+
+  <name>OpenSwitch Added Debian Stretch packages</name>
+  <type>distro</type>
+  <default_solver/>
+
+  <xi:include href="repo/stretch.xml"/>
```
Comment (heucke) [inline release_bp/opx_debian_base_stretch.xml:8]: @GarrickHe @jeff-yin Still can't get bind9 from stretch-backports to install. We can go with no bind9 at all or use the regular bind9 (no backports). I opted for regular bind9 in this PR.

**Comment by heucke** [comment]
Pull-request updated, HEAD is now b3ac1860a9c4a37890a05b2c5984c166bff92a5a

**Comment by heucke** [comment]
@GarrickHe It's now failing at packages from the sync which haven't been published yet. The kernel is pinned to `4.9.82-1+deb9u3`. You can use my fork to test as we get packages from the sync published. Once all packages are published, this PR will build successfully and I can merge it.

**Comment by GarrickHe** [comment]
@theucke No problem. I'll give it a try as soon as Howard puts up the new opx-platform-config packages.

**Comment by heucke** [comment]
Pull-request updated, HEAD is now a487536bb8dea899f740462ab43c7e0f866e48a6

**Comment by heucke** [comment]
Need to update `README.md` with new blueprint for Stretch.

**Comment by heucke** [comment]
Pull-request updated, HEAD is now 99c9017c42c8127c26c5b93a2814a7abb265f6a6

**Comment by heucke** [comment]
@jeff-yin I added `ftp` and `ntp` only to the Stretch blueprint.

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
Sorry, remove ftp. Spoke to @jghalam and he doesn't recommend adding FTP to the installer by default.

**Comment by heucke** [comment]
Pull-request updated, HEAD is now 4bb37f808f7ef74c79500e5dee92706bc586aa6e

**Comment by heucke** [comment]
Amended my commit, only leaving ntp.

**Review by jeffyin-dell — DISMISSED** [review]

**Comment by heucke** [comment]
Pull-request updated, HEAD is now 1b0fe1362613de1970db31571c8411031b1ab4bf

**Comment by heucke** [comment]
Pull-request updated, HEAD is now 7af252db5a7b4c43db6a2cf757b2b8408971a9ce

**Comment by heucke** [comment]
@jeff-yin I've set it up so that merging this PR will publish a Stretch Unstable installer to http://archive.openswitch.net/installers/unstable/Dell-EMC/PKGS_OPX-unstable-stretch-installer-x86_64.bin

**Review by jeffyin-dell — APPROVED** [review]

**Pull request merged**